include/tester/vtcp/tcp/TCPServer.hpp

```diff
@@ -56,7 +57,7 @@ class TCPServer
 
     void onMessageReceived(const std::size_t id, const Message& request) 
     {
-        _requestHandler->handle(_sessions.at(id), request);
+        _sessions.at(id)->handle(request);
```

Comment:
wrap this code with try / catch and print to `std::cerr` if there's no session with this ID.

Reply:
done